helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -216,6 +216,9 @@ alfresco-deployment-service:
     name: postgresql-ads
     username: alfresco
     password: alfresco
+  application:
```

Comment:
@SohelSaiyed maybe for consistency with var name i would use plural "applications" ?